x-pack/plugins/security_solution/server/lib/detection_engine/fleet_integrations/api/get_installed_integrations/installed_integration_set.ts

```diff
@@ -71,6 +80,39 @@ export const createInstalledIntegrationSet = (): IInstalledIntegrationSet => {
     }
   };
 
+  const addInstalledPackage = (installedPackage: PackageListItem): void => {
+    const packageInfo = {
+      package_name: installedPackage.name,
+      package_title: installedPackage.title,
+      package_version:
+        // Actual `installed_version` is buried in SO, root `version` is latest package version available
+        // SO should be available as PackageListItem should be InstalledRegistry<RegistrySearchResult>
+        // @ts-expect-error Property 'savedObject' does not exist on type 'PackageListItem'
```

Comment:
Should be able to type narrow this to `InstalledRegistry<RegistrySearchResult>` when filtering packages on installed state, but not entirely sure why it isn't working as-is since `PackageList` = `Installable<RegistrySearchResult>` and `Installable<T>` itself is an `|` of `InstalledRegistry<T>` which actually has the `savedObject` field...